DotaPlayerData.UI/Pages/SearchResultGrid.razor

```diff
@@ -0,0 +1,27 @@
+﻿@using DotaPlayerData.Core
+
+<div>
+    @if (SearchPlayerResults != null)
+    {
+        <MudList Clickable="true" @bind-SelectedItem="selectedItem" Color="Color.Success">
+            @foreach (var player in SearchPlayerResults)
+            {
+                <MudListItem Value="player.AccountId" Class="player-list-item">
+                    <MudStack Row="true">
+                    <MudImage Src="@player.AvatarFull" Class="rounded-lg" Width="50" Height="50" Elevation="25"/>
+                    <MudText>@player.PersonaName</MudText>
+                    </MudStack>
+                </MudListItem>
```

Comment:
**Suggestion:** Consider adding a click event handler to the MudListItem to handle user interactions.
```suggestion
                <MudListItem Value="player.AccountId" Class="player-list-item" OnClick="() => OnPlayerClicked(player)">
                    <MudStack Row="true">
                    <MudImage Src="@player.AvatarFull" Class="rounded-lg" Width="50" Height="50" Elevation="25"/>
                    <MudText>@player.PersonaName</MudText>
                    </MudStack>
                </MudListItem>

                @code {
                    // ... existing code ...

                    private void OnPlayerClicked(SearchPlayerResult player)
                    {
                        // Handle the click event
                    }
                }
```